deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -156,6 +156,13 @@ Resources:
             AllowedOrigins:
               - '*'
             MaxAge: 3000
+          - Id: CSRFCorsHeader
+            AllowedHeaders:
+              - X-CSRF-Token
+            AllowedMethods:
```

Comment:
Can we add HEAD and OPTIONS? Or are those implicitly a subset of GET?

Reply:
`OPTIONS` is not supported by S3 configurations - we can add `HEAD` without too much of a problem. See: [AWS S3 CloudFormation Reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-cors-corsrule.html#cfn-s3-bucket-cors-corsrule-allowedmethods)